day01/day01.py

```diff
@@ -1,50 +1,80 @@
-def main(inputFile): 
-    with open(inputFile,'r') as myfile: 
+def main(inputFile):
+    with open(inputFile, "r") as myfile:
         inputList = myfile.readlines()
-    print(inputList)
+    # print(inputList)
     print(part1(inputList))
     print(part2(inputList))
 
 
 def part1(inputList):
     valueList = []
-    for line in inputList: 
+    for line in inputList:
         first = firstDigit(line)
         last = lastDigit(line)
-        #print(first, last)
-        valueList.append(int(str(first)+str(last)))
+        # print(first, last)
+        valueList.append(int(str(first) + str(last)))
     return sum(valueList)
 
+
 def part2(inputList):
-    searchterms = {'zero': 0, 'one': 1, 'two': 2, 'three': 3, 'four': 4, 'five': 5, 'six': 6, 'seven': 7, 'eight': 8, 'nine': 9,
-                    '0': 0, '1': 1, '2': 2, '3': 3, '4': 4, '5': 5, '6': 6, '7': 7, '8': 8, '9': 9}
+    searchterms = {
+        "zero": 0,
+        "one": 1,
+        "two": 2,
+        "three": 3,
+        "four": 4,
+        "five": 5,
+        "six": 6,
+        "seven": 7,
+        "eight": 8,
+        "nine": 9,
+        "0": 0,
+        "1": 1,
+        "2": 2,
+        "3": 3,
+        "4": 4,
+        "5": 5,
+        "6": 6,
+        "7": 7,
+        "8": 8,
```

Comment:
löschen, dann direkt drunter:

`for i in range(9): dict.update({str(i+1):i+1})`